### ISSUE TITLE
Exclude your visits from Google Analytics (on production)

### ISSUE THREAD

**Issue by trinhngocthuyen** [trinhngocthuyen/TIL#15]
While disabling *google analytics* in Debug/Staging build is just a piece of cake, it's not that easy to exclude your GA traffic from your own visits when it's live.

[This post](http://www.daniloaz.com/en/5-ways-to-exclude-your-own-visits-from-google-analytics/) suggested 5 approaches to alleviate this issue. Yet, I found 2 of them suitable for my needs:

**[1] Filter IP address in Google analytics:**
- This approach is easy. Nothing to do with the code. We just have to create some *filters* in Google Analytics settings.
- *Drawbacks*: suitable for fixed public IPs (for example, you mostly access your website from your home or office). The report could still be messed up if you keep visiting your site when using 3G, or in a coffee house.

**[2] Use this chrome extension** [Block yourself from Analytics](https://chrome.google.com/webstore/detail/block-yourself-fm-analy/fadgflmigmogfionelcpalhohefbnehm)
- I love this way 😆 since It works perfectly with dynamic IPs, and I have to do nothing but installing the extension.
- Basically, the core idea behind this tool is to block the app from contacting with Google Analytics domains. Therefore, you can come up with an alternative: disable `www.google-analytics.com` in etc/hosts 😄.
- *Drawbacks*: the tool is not available for Safari or mobile devices. P/s: this limitation does not bother me much since I barely visit my blog from mobile devices.
